RFCs/0035-partiql-window-function.md

```diff
@@ -0,0 +1,1224 @@
+    p : [
+          <stock : {'trade_date': 2022-09-30, 'ticker': 'GOOG', 'price': 96.15}>
+        ],
+    pos: 0
+   > 
+>>
+```
+
+Example 1.2.5: Interaction with GROUP BY:
+
+```
+GROUP BY EXTRACT(MONTH FROM trade_date) as month, ticker GROUP AS g
+WINDOWED (PARTITION BY stock.ticker ORDER BY month) PARTITION AS p AT pos
+```
+
+First, consider the output binding collection from GROUP BY: 
+
+```
+<<
+  <
+    'month': 9, 'ticker': 'AMZN',
+    'g': <<
+      <
+        'stock': { 'trade_date': `2022-09-30`, 'ticker': 'AMZN', 'price': 113.00 }
+      >
+    >>
+  >,
+  <
+    'month': 9, 'ticker': 'GOOG',
+    'g': <<
+      <
+        'stock': { 'trade_date': `2022-09-30`, 'ticker': 'GOOG', 'price': 96.15}
+      >
+    >>
+  >,
+  <
+    'month': 10, 'ticker': 'AMZN',
+    'g': <<
+      <
+        'stock': { 'trade_date': `2022-10-03`, 'ticker': 'AMZN', 'price': 115.88 }
+      >
+    >>
+  >
+>>
+```
+
+Next, consider the output binding collection FROM WINDOWED clause: 
+```
+<<
+  <
+    'month': 9, 'ticker': 'AMZN',
+    'g': <<
+      <
+        'stock': { 'trade_date': `2022-09-30`, 'ticker': 'AMZN', 'price': 113.00 }
+      >
+    >>
+    p : [
+          < 
+            'month': 9, 'ticker': 'AMZN',
+            'g': << 
+              <
+                'stock': { 'trade_date': `2022-09-30`, 'ticker': 'AMZN', 'price': 113.00 }
+              >
+            >>
+          >, 
+          < 'month': 10, 'ticker': 'AMZN',
+             'g': <<
+               <
+                 'stock': { 'trade_date': `2022-10-03`, 'ticker': 'AMZN', 'price': 115.88 }
+               >
+             >>
+          >
+        ],
+    pos: 0
+  >,
+  <
+    'month': 9, 'ticker': 'GOOG',
+    'g': <<
+              <
+                'stock': { 'trade_date': `2022-09-30`, 'ticker': 'GOOG', 'price': 96.15 }
+              >
+         >>
+    p : [ 
+          <
+            'month': 9, 'ticker': 'GOOG',
+            'g': <<
+                   <
+                    'stock': { 'trade_date': `2022-09-30`, 'ticker': 'GOOG', 'price': 96.15 }
+                   >
+                  >>
+          >        
+        ],
+    pos : 0    
+  >,
+  <
+    'month': 10, 'ticker': 'AMZN',
+    'g': <<
+      <
+        'stock': { 'trade_date': `2022-10-03`, 'ticker': 'AMZN', 'price': 115.88 }
+      >
+    >>
+    p : [
+          < 'month': 9, 'ticker': 'AMZN',
+            'g': << 
+              <
+                'stock': { 'trade_date': `2022-09-30`, 'ticker': 'AMZN', 'price': 113.00 }
+              >
+            >>
+          >, 
+          < 'month': 10, 'ticker': 'AMZN',
+             'g': <<
+               <
+                 'stock': { 'trade_date': `2022-10-03`, 'ticker': 'AMZN', 'price': 115.88 }
+               >
+             >>
+          >
+        ],
+    pos: 1
+  >
+>>
+```
+
+
+Example 1.2.6: Data set with NULL and MISSING: 
+
+Recall that the equivalence function will treat the **NULL** and the **MISSING** expression as the same: 
```

Comment:
It's good that we're referring to equivalence class here but let's also add another note explaining the behavior based on the equivalence rule for `NULL` and `MISSING`; like the following quote from section `11.1.1`:
```
Notice that both the 3rd and 5th tuples of logs were grouped under the sensor:null group, despite the sensor of
the 3rd being NULL while the sensor of the 5th being MISSING. The query result is
```

Reply:
Addressed in the new commit.